src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue11655.cs

```diff
@@ -0,0 +1,23 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue11655 : _IssuesUITest
+	{
+		public Issue11655(TestDevice device) : base(device) { }
+
+		public override string Issue => "Label's HorizontalTextAlignment property is not updated properly at runtime";
+
+		[Test]
+		[Category(UITestCategories.Label)]
+		[FailsOnMac]
```

Comment:
Please don't use this attribute

Just if/def out this file and log an issue that the test fails on MAC

Reply:
I have removed that attribute.